srcpkgs/cbatticon-gtk3/template

```diff
@@ -29,6 +30,10 @@ do_build() {
 		make ${makejobs} ${make_build_args} ${make_build_target}
 }
 
+post_install() {
+	vlicense COPYING
```

Comment:
There is no need to include a copy of GPL-2. It is already installed on every void system under `/usr/share/licenses/GPL-2`

Reply:
OK, thanks. Should be solved now.